src/Monolog/Logger.php

```diff
@@ -285,9 +285,11 @@ public function addRecord($level, $message, array $context = array())
         foreach ($this->processors as $processor) {
             $record = call_user_func($processor, $record);
         }
-        while (isset($this->handlers[$handlerKey]) &&
```

Comment:
the old code let all loggers - starting at the first which supports the current error level - handle the logmessage as long as they `return false`..?

this in turn is returns `false` as long as your handler defines `bubble=true`.
not sure what is missing to get it running for your case?